content/en/news/releases/1.15.x/announcing-1.15.1/index.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Announcing Istio 1.15.1
+linktitle: 1.15.1
+subtitle: Patch Release
+description: Istio 1.15.1 patch release.
+publishdate: 2022-09-23
+release: 1.15.1
+---
+
+This release contains bug fixes to improve robustness.
+
+This release note describes what is different between Istio 1.15.0 and Istio 1.15.1.
+
+{{< relnote >}}
+
+## Changes
+
+- **Added** `PILOT_ENABLE_K8S_SELECT_WORKLOAD_ENTRIES` feature back to Istio which was removed in 1.14. Will
+persist until the use case is clarified and more permanent API added.
+
+- **Fixed** an issue where `AddRunningKubeSourceWithRevision` returns an error causing the Istio Operator
+to go into an error loop. ([Issue #39599](https://github.com/istio/istio/issues/39599))
+
+- **Fixed** an issue where adding a `ServiceEntry` could affect an existing `ServiceEntry` with the same hostname.
+([Issue #40166](https://github.com/istio/istio/issues/40166))
+
+- **Fixed** an issue where user can not delete iop resource with revision if istiod is not running.
```

Comment:
instead of putting "iop" in spelling, make this say "IstioOperator"

Reply:
Done